app/models/concerns/rad_user.rb

```diff
@@ -74,7 +70,9 @@ module RadUser
     validate :validate_twilio_verify
     validate :validate_mobile_phone
     validate :password_excludes_name
-    validates :security_roles, presence: true, if: :active?
+
+    # this should be changed to "if: :active?" at some point, see Task 2024
+    validates :security_roles, presence: true, if: :active_for_authentication?
```

Comment:
had to change this due to an existing "bug" that allows inviting a user w/o a role, we had an old task for that